apps/roam/src/components/settings/DiscourseNodeSuggestiveRules.tsx

```diff
@@ -64,10 +68,56 @@ const DiscourseNodeSuggestiveRules = ({
     (e: React.ChangeEvent<HTMLInputElement>) => {
       const newValue = e.target.value;
       setEmbeddingRef(newValue);
+      node.embeddingRef = newValue;
     },
-    [],
+    [node],
   );
-
+  const [isUpdating, setIsUpdating] = useState(false);
+
+  const handleUpdateEmbeddings = async (): Promise<void> => {
+    setIsUpdating(true);
+    try {
+      const blockNodesSince = await getAllDiscourseNodesSince(
+        new Date(0).toISOString(),
+        [node],
+      );
+      const supabaseClient = await getLoggedInClient();
+      if (!supabaseClient) return;
+
+      const context = await getSupabaseContext();
+      if (context && blockNodesSince) {
+        await upsertNodesToSupabaseAsContentWithEmbeddings(
+          blockNodesSince,
+          supabaseClient,
+          context,
+        );
+        const nodeBlockToLocalConcepts = blockNodesSince.map((node) => {
```

Comment:
Not sure why a change of content needs to trigger a change of concept here? I suspect the whole stanza is unneeded.